<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Project6883Class</class>
 <widget class="QWidget" name="Project6883Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1030</width>
    <height>729</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Financial Computing Project</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <widget class="QPushButton" name="okButton">
     <property name="text">
      <string>Compute</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="font">
        <font>
         <family>Arial Unicode MS</family>
        </font>
       </property>
       <property name="text">
        <string>Threshold:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="thresholdEdit">
       <property name="text">
        <string>10</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>% (5 -10).           </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_3">
       <property name="whatsThis">
        <string>How many times we choose 40 stocks from each group to calculate AAR and CAAR</string>
       </property>
       <property name="text">
        <string>Bootstraping Times:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="BootstrapingEdit">
       <property name="whatsThis">
        <string>How many times we choose 40 stocks from each group to calculate AAR and CAAR</string>
       </property>
       <property name="text">
        <string>1</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>1</number>
     </property>
     <widget class="QWidget" name="stocksTab">
      <attribute name="title">
       <string>Stocks</string>
      </attribute>
      <widget class="QListWidget" name="stockInformationList">
       <property name="geometry">
        <rect>
         <x>476</x>
         <y>48</y>
         <width>471</width>
         <height>551</height>
        </rect>
       </property>
      </widget>
      <widget class="QTableWidget" name="stocksTable">
       <property name="geometry">
        <rect>
         <x>11</x>
         <y>48</y>
         <width>461</width>
         <height>551</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>Date</string>
        </property>
        <property name="toolTip">
         <string comment="1"/>
        </property>
        <property name="whatsThis">
         <string/>
        </property>
        <property name="font">
         <font>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Price</string>
        </property>
        <property name="toolTip">
         <string extracomment="2"/>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Daily Return</string>
        </property>
        <property name="whatsThis">
         <string>3</string>
        </property>
       </column>
      </widget>
      <widget class="QPushButton" name="searchButton">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>0</y>
         <width>112</width>
         <height>34</height>
        </rect>
       </property>
       <property name="text">
        <string>search</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_4">
       <property name="geometry">
        <rect>
         <x>1</x>
         <y>10</y>
         <width>63</width>
         <height>18</height>
        </rect>
       </property>
       <property name="text">
        <string>Ticker:</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="stockEdit">
       <property name="geometry">
        <rect>
         <x>71</x>
         <y>6</y>
         <width>193</width>
         <height>24</height>
        </rect>
       </property>
      </widget>
      <widget class="QListWidget" name="listWidget">
       <property name="geometry">
        <rect>
         <x>71</x>
         <y>30</y>
         <width>191</width>
         <height>192</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="groupsTab">
      <attribute name="title">
       <string>Groups</string>
      </attribute>
      <widget class="QLabel" name="label_5">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>9</y>
         <width>111</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Choose Group:</string>
       </property>
      </widget>
      <widget class="QComboBox" name="groupComboBox">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>0</y>
         <width>151</width>
         <height>31</height>
        </rect>
       </property>
       <property name="currentText">
        <string/>
       </property>
       <item>
        <property name="text">
         <string/>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Beat</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Meet</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Miss</string>
        </property>
       </item>
      </widget>
      <widget class="QTableWidget" name="groupTable">
       <property name="geometry">
        <rect>
         <x>9</x>
         <y>35</y>
         <width>301</width>
         <height>581</height>
        </rect>
       </property>
       <column>
        <property name="text">
         <string>Ticker</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>Surprise</string>
        </property>
       </column>
      </widget>
      <widget class="QPushButton" name="excelButton">
       <property name="geometry">
        <rect>
         <x>320</x>
         <y>580</y>
         <width>691</width>
         <height>39</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Arial Unicode MS</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>Plot in Excel</string>
       </property>
      </widget>
      <widget class="QCustomPlot" name="plot" native="true">
       <property name="geometry">
        <rect>
         <x>318</x>
         <y>10</y>
         <width>671</width>
         <height>561</height>
        </rect>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>2</width>
         <height>2</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout"/>
      </widget>
      <zorder>layoutWidget</zorder>
      <zorder>label_5</zorder>
      <zorder>groupComboBox</zorder>
      <zorder>groupTable</zorder>
      <zorder>excelButton</zorder>
      <zorder>plot</zorder>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
